docs/getting-started/linux-standalone.md

```diff
@@ -0,0 +1,170 @@
+
+### Other
+
+The remaining installation methods support environmental variables to configure
+the Collector. The following environmental variables are required:
+
+- `SPLUNK_REALM` (no default): Which realm to send the data to (for example: `us0`)
+- `SPLUNK_ACCESS_TOKEN` (no default): Access token to authenticate requests
+- `SPLUNK_MEMORY_TOTAL_MIB` (no default): Total memory allocated to the Collector.
+
+<details>
+<summary>
+Optional environment variables
+</summary>
+
+- `SPLUNK_CONFIG` (default = `/etc/otel/collector/splunk_config_linux.yaml`): Which configuration to load.
+- `SPLUNK_BALLAST_SIZE_MIB` (no default): How much memory to allocate to the ballast.
+- For Linux systems:
+  - `SPLUNK_MEMORY_LIMIT_PERCENTAGE` (default = `90`): Maximum total memory to be allocated by the process heap.
+  - `SPLUNK_MEMORY_SPIKE_PERCENTAGE` (default = `20`): Maximum spike between the measurements of memory usage.
+- For non-Linux systems:
+  - `SPLUNK_MEMORY_LIMIT_MIB` (no default): Maximum total memory to be allocated by the process heap.
+  - `SPLUNK_MEMORY_SPIKE_MIB` (no default): Maximum spike between the measurements of memory usage.
+
+> `SPLUNK_MEMORY_TOTAL_MIB` automatically configures the ballast, memory limit,
+> and memory spike. If the optional environment variables are defined, they
+> will override the value calculated from `SPLUNK_MEMORY_TOTAL_MIB`.
+</details>
+
+### Docker
+
+Deploy from a Docker container. Replace `0.1.0` with the latest stable version number:
+
+```bash
+$ docker run --rm -e SPLUNK_ACCESS_TOKEN=12345 -e SPLUNK_MEMORY_TOTAL_MIB=1024 \
+    -e SPLUNK_REALM=us0 -p 13133:13133 -p 14250:14250 -p 14268:14268 -p 55678-55680:55678-55680 \
+    -p 6060:6060 -p 7276:7276 -p 8888:8888 -p 9411:9411 -p 9943:9943 \
+    --name otelcol quay.io/signalfx/splunk-otel-collector:0.1.0
+```
+
+### Standalone
+
+Run as a binary on the local system:
+
+```bash
+$ git clone https://github.com/signalfx/splunk-otel-collector.git
+$ cd splunk-otel-collector
+$ make install-tools
+$ make otelcol
+$ SPLUNK_REALM=us0 SPLUNK_ACCESS_TOKEN=12345 SPLUNK_MEMORY_TOTAL_MIB=1024 \
+    ./bin/otelcol
+```
+
+## Advanced Configuration
+
+### Command Line Arguments
+
+After the binary command or Docker container command line arguments can be
+specified.
+
+> IMPORTANT: Command line arguments take precedence over environment variables.
+
+For example in Docker:
+
+```bash
+$ docker run --rm -e SPLUNK_ACCESS_TOKEN=12345 -e SPLUNK_MEMORY_TOTAL_MIB=1024 \
+    -e SPLUNK_REALM=us0 -p 13133:13133 -p 14250:14250 -p 14268:14268 -p 55678-55680:55678-55680 \
+    -p 6060:6060 -p 7276:7276 -p 8888:8888 -p 9411:9411 -p 9943:9943 \
+    --name otelcol quay.io/signalfx/splunk-otel-collector:0.1.0 \
+        --log-level=DEBUG
+```
+
+> Use `--help` to see all available CLI arguments.
+
+### Custom Configuration
+
+In addition to using the default configuration, a custom configuration can also
```

Comment:
nit, and know this wasn't introduced here, but this can sound like both the default and provided configs are being used instead of just the provided one.